code/modules/mob/living/carbon/human/species.dm

```diff
@@ -965,7 +965,7 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 		H.losebreath = 0
 
 		var/takes_crit_damage = (!HAS_TRAIT(H, TRAIT_NOCRITDAMAGE))
-		if((H.health < H.crit_threshold) && takes_crit_damage)
+		if((H.health < H.crit_threshold) && takes_crit_damage && H.stat != DEAD)
```

Comment:
this should happen sooner. human/Life() chain processes heart and liver damage before `spec_life()` procs, so `H.stat != DEAD` should probably be checked at 963 instead since they can die before it happens.